app/ui/src/app/customizations/api-connector/api-connector-list.component.ts

```diff
@@ -24,16 +26,19 @@ const category = getCategory('ApiConnectors');
   styleUrls: ['./api-connector-list.component.scss']
 })
 export class ApiConnectorListComponent implements OnInit {
-  connectors: Observable<Connectors>;
-  filteredConnectors: Subject<Connectors> = new BehaviorSubject(<Connectors>{});
-  loading: Observable<boolean>;
+  apiConnectors$: Observable<ApiConnectors>;
+  filteredApiConnectors$: Subject<ApiConnectors> = new BehaviorSubject(<ApiConnectors>{});
+  loading$: Observable<boolean>;
   listConfig: ListConfig;
 
   constructor(
     public config: ConfigService,
-    private store: ConnectorStore,
+    private store: ApiConnectorStore,
```

Comment:
I'd suggest to name the class member as `apiConnectorStore` instead of just `store`. This is a common practice borrowed from ES5 and ES6, where there is no type inference and we resource to the variable/field member to inform about its type. This also ensures not name collisions will occur whenever we inject another WhateverStore in our component. Eg: A component that pulls information from both the `ApiConnectorStore` (for connector details) and a future `SyndesisI18NStore` (for localization)...

Reply:
Good point!